dags/pegs-challenge-submission-dag.py

```diff
@@ -11,21 +11,20 @@
 from orca.services.nextflowtower.models import LaunchInfo
 from orca.services.synapse import SynapseHook
 
-UUID = uuid.uuid4()
 REGION_NAME = "us-east-1"
 BUCKET_NAME = "pegs-challenge-project-tower-scratch"
-FILE_NAME = f"submissions_{UUID}.csv"
+FILE_NAME = "submissions.csv"
 KEY = "10days/pegs_challenge"
 
 dag_params = {
     "synapse_conn_id": Param("SYNAPSE_ORCA_SERVICE_ACCOUNT_CONN", type="string"),
-    "synapse_evaluation_id": Param("9615511", type="string"),
     "aws_conn_id": Param("AWS_TOWER_PROD_S3_CONN", type="string"),
+    "revision": Param("e19ca1dab3a85d77b62e8e00481d7291c19a0048", type="string"),
```

Comment:
Is there documentation around where this revision string is coming from?

Reply:
No, but that's a good point. I don't think there is any comprehensive piece of documentation outlining how `orca-recipes` and `nf-synapse-challenge` go together. I do have a ticket with a scope that this falls into, and I'll link this thread convo to that ticket so I don't forget to talk about params.

To keep it brief, this basically points to a branch, a version, or a commit ID of the pipeline that executes the NF workflow (in this case, `nf-synapse-challenge`)

Reply:
The parameter could also be renamed something more descriptive so its clear what it is referring to